app/services/transaction_search/ordered_for_searcher.rb

```diff
@@ -0,0 +1,25 @@
+module TransactionSearch
+
+  class OrderedForSearcher < BaseSearcher
+
+    def options
+      User.select("users.id, users.first_name, users.last_name")
```

Comment:
Just curious, does this still work if you used symbols? `User.select(:id, :first_name, :last_name)`. I feel like that'll be a little safer (assuming it works).